code/datums/uplink_item.dm

```diff
@@ -147,6 +147,16 @@ var/list/uplink_items = list()
 	cost = 2
 	job = list("Chef")
 
+//Chaplain
+
+/datum/uplink_item/jobspecific/voodoo
+	name = "Voodoo Doll"
+	desc = "A doll created by Syndicate Witch Doctors. Ingridents:Something of the Thread,Something of the Head,Something of the Body,Something of the Dead and monosodium glutamate."
```

Comment:
alot of things in the PR have a reference to something...its non SS13 related....